.github/workflows/cicd.yaml

```diff
@@ -57,7 +56,9 @@ jobs:
       - run: |
           python -m build
       - uses: pypa/gh-action-pypi-publish@release/v1
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
Hi @JohnPaton 

You probably want to check the repository here,
otherwise a tag on the external PR repo might trigger the publication
(I'm guessing here)

``` yaml
if: github.repository == 'JohnPaton/airbase' && ...
```

Reply:
Ah nice, good catch. Still getting the hang of Actions.

Reply:
Ah it wouldn't succeed anyway, because forks don't have access to the secrets so they can't publish to PyPI. I've popped the extra condition in anyway in #22, to avoid any extra failures.